src/less/button/ds4/button-secondary.less

```diff
@@ -17,22 +17,22 @@ a.fake-btn--secondary {
     }
 }
 
-button.btn--secondary[disabled],
-button.expand-btn--secondary[disabled],
-button.btn--secondary[aria-disabled="true"],
-button.expand-btn--secondary[aria-disabled="true"] {
-    background-color: @ds4-color-core-gray-silver;
-
-    svg {
-        color: @ds4-color-core-gray-davys;
-    }
-
-    &:hover,
-    &:focus {
-        background-color: @ds4-color-core-gray-silver;
-        border-color: @ds4-color-text-disabled;
-    }
-}
+// button.btn--secondary[disabled],
```

Comment:
Intentionally commented out?

Reply:
Whoops. I thought I deleted that. Let me fix it.